dao/src/integ-test/java/uk/gov/moj/sdt/dao/BulkCustomerDaoTest.java

```diff
@@ -49,13 +48,14 @@
 import uk.gov.moj.sdt.test.utils.TestConfig;
 
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.List;
 import javax.persistence.criteria.CriteriaBuilder;
 import javax.persistence.criteria.CriteriaQuery;
 import javax.persistence.criteria.Predicate;
 import javax.persistence.criteria.Root;
 
+import static org.junit.Assert.*;
```

Comment:
Replace this with the explicit static JUnit imports needed by this class (assertNotNull and assertEquals).